.github/workflows/contract-tests.yml

```diff
@@ -28,10 +28,11 @@ jobs:
           node-version: 16
       - name: Install dependencies
         run: cd ../../ && yarn install --pure-lockfile
-      - name: Compile contracts-versions
-        run: cd ../contracts-versions && yarn build
-      - name: Remove directories
-        run: rm -rf artifact cache deployments typechain
+      - name: Build the osx-versions NPM package
+        run: yarn build
+        working-directory: packages/contracts-versions/
+      - name: Clean the contracts directory
+        run: yarn clean
```

Comment:
Shouldn't this also include `working-directory: packages/contracts-versions/` or is it intentionally targeting the contracts folder?

Reply:
It is intentionally targeting the contracts folder. It could be better to explicitly add the working directory.
We want to be 100% sure that there is no old artifact is left from the previous compilation or checking out old commits.